<launch>
   <arg name="vehicleX" default="0.0" />
   <arg name="vehicleY" default="0.0" />
   <arg name="vehicleYaw" default="0.0" />
   <arg name="ns" default="" />

   <node pkg="map_to_odom" type="map_to_odom" name="map_to_odom" output="screen" respawn="true">
      <param name="vehicleX" type="double" value="$(arg vehicleX)" />
      <param name="vehicleY" type="double" value="$(arg vehicleY)" />
      <param name="vehicleYaw" type="double" value="$(arg vehicleYaw)" />
      <param name="ns" type="string" value="$(arg ns)" />
   </node>

</launch>